03/app.js

```diff
@@ -5,14 +5,53 @@ const a = randomNumber(min, max);
 const b = randomNumber(min, max);
 const c = randomNumber(min, max);
 
-console.log(a, b, c);
-
-
-
-
-
-
+// console.log(a, b, c);
 
 function randomNumber(min, max) {
     return Math.round((Math.random() * (max - min)) + min);
-}
\ No newline at end of file
+}
+
+function getSum(a, b, c) {
+    a = parseInt(a);
+    b = parseInt(b);
+    c = parseInt(c);
+    const arr = [a, b, c]
+    arr.sort(function(a, b){return b-a})
+    let first = arr[0];
+    let second = arr[1];
+    let result = first + second;
+    return result;
+} 
+function isEven(sum) {
+    if (isNaN(sum)) {
+        return null
+    }
+    if (!isNaN(sum) && sum % 2 === 0) {
+        return true;
```

Comment:
Można też:
```
} else if (sum % 2 === 0) {
```

czyli pierwszy warunek  tj. `isNaN()`  sparwdza czy jest `NaN` - jeśli jesteśmy w drugim warunku to już wiemy że nie jest ;) (dlatego `else`)